internal/delivery/http/tasks/readone/readone.go

```diff
@@ -0,0 +1,81 @@
+package readone
+
+import (
+	"context"
+	"log/slog"
+	"net/http"
+	"reflect"
+	"strconv"
+
+	"github.com/10Narratives/task-tracker/internal/models"
+	"github.com/go-chi/render"
+)
+
+// TODO: Написать документацию для структуры ответа и интерфейса
+// TODO: Написать документацию для обработчика с помощью Swagger
+
+const op = "http.Readone"
```

Comment:
Кажется так более читаемо будет

```suggestion
const op = "http.ReadOne"
```